sotodlib/io/load_smurf.py

```diff
@@ -2085,10 +2085,13 @@ def load_file(filename, channels=None, ignore_missing=True,
     if len(filenames) == 0:
         logger.error("No files provided to load")
     
-    if status is None:
-        try:    
-            logger.warning('Loading status frame from the file at the start'
+    if status is None or status.num_chans is None:
+        try:
+            if status is None:
+                logger.warning('Loading status frame from the file at the start'
                             ' of the corresponding observation.')
+            else:
+                logger.warning("Status information is missing 'num_chans.' Trying to fix.")
```

Comment:
Idea is just to make sure user is aware that this key is missing from the status?

Reply:
yeah. Idea is to give the actual error instead of one that might point them in the wrong direction. Big thing is that fixing this error _well_ will take me awhile.